server/db/index.js

```diff
@@ -1,58 +1,71 @@
 const db = require('./db');
 
 //import all models
-const Order = require('./models/Order');
-const OrderDrink = require('./models/Order_Drink');
-const Menu = require('./models/Menu');
-const Employee = require('./models/Employee');
 const Drink = require('./models/Drink');
-const Tab = require('./models/Tab');
+const Employee = require('./models/Employee');
+const Menu = require('./models/Menu');
+const Order = require('./models/Order');
+const OrderDrink = require('./models/OrderDrink');
 const PickUpLocation = require('./models/PickUpLocation');
-const Venue = require('./models/Venue');
-const User = require('./models/User');
+const Tab = require('./models/Tab');
 const TabDrink = require('./models/TabDrink');
-
-//model associations
+const User = require('./models/User');
+const Venue = require('./models/Venue');
```

Comment:
Alphabetized these so we can quickly confirm everything is there by comparing to file structure